main/main.c

```diff
@@ -33,6 +34,10 @@ int main(int argc, char *argv[]){
         (void)scanf("%d", &sleep_seconds);   
         ret_code = my_sleep(sleep_seconds);
     }
+    else if (strcmp( command, "my_uptime" ) == 0){
```

Comment:
line 37 should compare to an integer no. The integer in the menu option that corresponds to "my_uptime".